features/media-import.feature

```diff
@@ -69,6 +69,25 @@ Feature: Manage WordPress attachments
       My fabulous caption
       """
 
+  Scenario: Import a file as attachment from a local image and leave it in it's current location
+    Given download:
```

Comment:
After `Given download`, you'll also need to run:

```
And I run `wp option update uploads_use_yearmonth_folders 0`
```

Otherwise, your file path assertion below will always be invalid, because WordPress will always used date-based folders.